### PR TITLE
fix: initial changes who we are page

### PR DESCRIPTION
**Pull request by mayuran-deriv**
Changes:

1. content changes for who we are page , and strapi update ,
2. hong kong office added to map

Screenshot 2023-09-28 at 4 18 55 PM

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]

File: src/pages/who-we-are/_EmployerAwards.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                        mb="40x"
+                        md={{ direction: 'row', mt: '30x', gap: '30x' }}
+                    >
+                        <div className={image_wrapper}>
+                            <StaticImage
+                                src="../../images/common/who-we-are/uf-awards.png"
+                                loading="eager"
+                                formats={['avif', 'webp', 'auto']}
+                                alt="banner"
+                            />
```
Comment (hasan-deriv) [inline src/pages/who-we-are/_EmployerAwards.tsx:40]:
```suggestion
                            <StaticImage
                                src="../../images/common/who-we-are/uf-awards.png"
                                formats={['avif', 'webp', 'auto']}
                                placeholder="blurred"
                                alt="banner"
                            />
```

Comment (hasan-deriv) [reply src/pages/who-we-are/_EmployerAwards.tsx:40]: if it has a fixed width and height for all devices than add `layout="fixed"` and give the **width** and **height**

**Review by hasan-deriv — CHANGES_REQUESTED** [review]

File: src/pages/who-we-are/_EmployerAwards.tsx (hunk 1 above)
Comment (jia-deriv) [reply src/pages/who-we-are/_EmployerAwards.tsx:40]: Do we need to have translation for the alternate text?

File: src/pages/who-we-are/_EmployerAwards.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                        </div>
+                        <Flex.Box gap="0x" direction="col">
```
Comment (aswathy-deriv) [inline src/pages/who-we-are/_EmployerAwards.tsx:42]: `gap='0x'` is needed?

File: src/pages/who-we-are/_types.ts [hunk 3]
```diff
@@ -1,3 +1,16 @@
+import { ReactElement } from 'react'
+import { TString } from 'types/generics'
+
+export type EmployerAwardsType = {
+    id: number
+    title: TString
+    text_with_link: {
```
Comment (aswathy-deriv) [inline src/pages/who-we-are/_types.ts:7]: Can we remove the `with` and the name to`text_link` instead of `text_with_link`

**Comment by aswathy-deriv** [comment]
Hi @mayuran-deriv ,Can you elaborate the description of the PR and add a list or bullets.Thanks

**Review by hasan-deriv — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

File: src/features/components/templates/footer/link-item/index.tsx [hunk 4]
```diff
@@ -18,7 +20,8 @@ const FooterLink = ({ item: { data } }: FooterLinkProps) => {
 
     return (
         <Link url={visible_url} size="small" pb="6x">
-            <Localize translate_text={data.text} />
+            {data?.text ? <Localize translate_text={data.text} /> : null}
+            {data?.src ? <Image src={data.src} className={footer_image} /> : null}
```
Comment (NikitK-deriv) [inline src/features/components/templates/footer/link-item/index.tsx:24]: conditional rendering is bad approach, let's create new template like `LinkImage`

File: src/pages/who-we-are/index.tsx [hunk 5]
```diff
@@ -50,6 +53,15 @@ const AboutUs = ({ data }: TWhoWeAre) => {
...
         banner,
     } = data?.strapiWhoWeArePage || {}
+
+    const clickToScrollHandler = useScrollToElement('investors-in-people')
+    const pathname_hash = getLocationHash()
+    useEffect(() => {
+        setTimeout(() => {
+            pathname_hash?.includes('investors-in-people') && clickToScrollHandler()
+        }, 1000)
+    }, [pathname_hash])
```
Comment (NikitK-deriv) [inline src/pages/who-we-are/index.tsx:63]: we have no extra re-renders here so we don't need a useEffect 
better to use this approach. please check the behaviour
```suggestion
        const pathname_hash = getLocationHash();
    const clickToScrollHandler = useScrollToElement('investors-in-people')

    if (pathname_hash?.includes('investors-in-people')) {
        clickToScrollHandler();
    }
```

**Pull request closed without merging**